plugins/chat/spec/system/navigation_spec.rb

```diff
@@ -26,4 +30,46 @@
       expect(page).to have_css(".chat-message-container[data-id='#{message.id}']")
     end
   end
+
+  context "when opening chat" do
+    it "opens the drawer by default" do
+      visit("/")
+      find(".open-chat").click
+
+      expect(page).to have_css(".topic-chat-container.expanded.visible")
+    end
+  end
+
+  context "when opening chat with full page as preferred mode" do
+    it "opens the full page" do
+      visit("/")
+      find(".open-chat").click
+      find(".topic-chat-drawer-header__full-screen-btn").click
```

Comment:
Unrelated to this PR, but if we haven't already planned to we should rename `topic-chat-X` classes to something else since we can't even make chats for topics anymore!

Reply:
yes we will do various renames now that chat is in core